packages/app/src/debug/DebugContainer.vue

```diff
@@ -117,4 +135,9 @@ const debugSpecsArray = computed(() => {
 
   return []
 })
+
+// TODO GH#24440 Re-map to use relevant run data point (currently stubbed with current run)
+const newerRelevantRun = computed(() => run.value)
+
+const isFirstPendingRun = computed(() => run.value && run.value.runNumber === 1 && run.value.status === 'RUNNING')
```

Comment:
Didn't think this through all the way (run numbers increment across all branches, so this will only work for the very first run on any branch for a given project). I think instead I'll need to grab the count of relevant runs from GQL and see if there's only one relevant run that happens to be pending

Reply:
I am working to encapsulate the logic in the server side to determine a `current` run (what the main part of the Debug page would show) and `newer` run (that could be `RUNNING` or a completed status).  Then the logic here for `isFirstPendingRun` would just be `currentRun.status === 'RUNNING'` or something like that.  And  the `newer` run would drive the `newerRelevantRun` variable

Reply:
@warrensplayer Sounds good, thanks for the context. Will leave this stubbed for now to ensure visibility & test coverage. If your backend additions merge first I'm happy to hook up as part of this PR, otherwise will plan to handle the hookup as follow-on changes